flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java

```diff
@@ -325,6 +325,7 @@ static void addVaadinDefaultsToJson(JsonObject json) {
         defaults.put("chokidar", "^3.5.0");
 
         defaults.put("idb", "5.0.6");
+        defaults.put("style-loader", "^2.0.0");
```

Comment:
Should probably use explicit version like in all the other versions here (except "chokidar", I'm not sure why that uses a caret range)?
```suggestion
        defaults.put("style-loader", "2.0.0");
```

Reply:
done

Reply:
>  I'm not sure why that uses a caret range
This has been added as a special case to fix a broken transitive dependency (chokidar 2). As we do not directly depend on it, the version of the compatible version was left as a range (see #8451).